### PR TITLE
access for triagers for project boards

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Based on this conversation: https://github.com/open-telemetry/community/issues/525#issuecomment-708877940

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
Also created ticket with GitHub asking why Write permissions are needed https://support.github.com/ticket/personal/0/872841

File: docs/how-to-configure-new-repository.md
Comment (arminru) [inline docs/how-to-configure-new-repository.md:31]: I think their approvals will still count for the minimum number of required approvals (by members with write access) and the code owner check will pass even if only one single code owner has approved.

Comment (SergeyKanzhelev) [reply docs/how-to-configure-new-repository.md:31]: yes, it still will be green. But at least some level of distinction between two roles may give some confidence to maintainers. Do you think some rewording is needed here? Or an additional note? 

I hope that GitHub can change this =)

Comment (arminru) [reply docs/how-to-configure-new-repository.md:31]: I just wanted to point this out in case it was missed but if there's no way to work around this restriction I think it's fine.

**Review by arminru — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**
